core/src/main/java/org/jdbi/v3/core/mapper/RowMapperFactory.java

```diff
@@ -33,4 +33,19 @@
      * @see ConfigRegistry#findColumnMapperFor(Type) for composition
      */
     Optional<RowMapper<?>> build(Type type, ConfigRegistry config);
+
+    /**
+     * Create a RowMapperFactory from a given {@link RowMapper} that matches
+     * a {@link Type} exactly.
+     *
+     * @param type the type to match with ==
+     * @param the mapper to return
+     *
+     * @return the factory
+     */
+    public static RowMapperFactory of(Type type, RowMapper<?> mapper) {
```

Comment:
This seems so obvious in retrospect.

Could we add this to `ColumnMapperFactory` as well, and add convenience methods registration methods for this pattern to `Configurable`, `RowMappers`, and `ColumnMappers`?